readme.md

```diff
@@ -88,6 +88,7 @@ Please raise a [Pull-Request](https://github.com/notthetup/awesome-webaudio/pull
 
 ### Books
 - [Web Audio API Book](http://chimera.labs.oreilly.com/books/1234000001552/index.html) - A book that brings the reader through the basics of WebAudio.
+- [JavaScript for Sound Artists](https://www.crcpress.com/JavaScript-for-Sound-Artists-Learn-to-Code-with-the-Web-Audio-API/Turner-Leonard/p/book/9781138961531) - A bottom up JavaScript / DOM course using Web Audio for all the examples
```

Comment:
Could you add a fullstop (period) at the end of the sentence. Sorry for being pedantic, but awesome requires it. :)